src/main/java/org/reactome/addlinks/IdentifiersDotOrgUtil.java

```diff
@@ -48,14 +48,15 @@
 	 * @param resourceIdentifier - The identifier of a resource, as per identifiers.org. If you are uncertain what the resourceIdentifier is,
 	 * you can search the identifiers.org website, or see a complete list of resources here:
 	 * <a href="https://identifiers.org/rest/resources">https://identifiers.org/rest/resources</a>
-	 * @return The accessURL of the resource identified by <code>resourceIdentifier</code>.
+	 * @return The accessURL of the resource identified by <code>resourceIdentifier</code>. In cases where identifiers.org does not return an accessURL, or where the
+	 * identifier is deactivated/deprecated (according to identifiers.org), NULL will be returned.
```

Comment:
Does the caller or callers of this method need to be updated to check for a NULL value?

Reply:
No. This is not new, only the comment that explicitly explains the NULL value is new.

Reply:
Okay, sounds good - I just wanted to check.